nightly/nightly.txt

```diff
@@ -14,6 +14,9 @@ pytest sanity/state_sync.py onetx 30
 pytest --timeout=600 sanity/state_sync.py onetx 250
 pytest --timeout=240 sanity/state_sync1.py
 pytest --timeout=310 sanity/state_sync2.py
+pytest --timeout=600 sanity/state_sync_routed.py notx 50
```

Comment:
50 is unlikely to test the state sync. Better to test at least 100.
Also no need to have all three modes in the nightly. Just `manytx` should be OK.

Reply:
done